fix: update vulnerable dependencies (#22)
Oneflow/handlebars-kit: README.md

> # NOTE: WIP FORK

Reviewer: ```suggestion
# Handlebars Kit

More than 150 helpers in ~15 categories designed to extend the Node.js Handlebars library.
```

> This is a fork of the [handlebars-kit](https://github.com/helpers/handlebars-helpers) created by [Jon Schlinkert](https://github.com/jonschlinkert).

Reviewer: ```suggestion
This is a fork of the [handlebars-helpers](https://github.com/helpers/handlebars-helpers) created by [Jon Schlinkert](https://github.com/jonschlinkert).
```

It's a fork of `handlebars-helpers` not `handlebars-kit`

> - The fork minimises dependencies to reduce the maintenance effort
> - The fork removes risky helpers, such as any that interact with the file system
> - The fork is published to NPM https://www.npmjs.com/package/handlebars-kit
> - The fork has higher test coverage

Reviewer: ```suggestion
Motivations for forking
- To minimise the number of dependencies, reducing the maintenance effort
- To remove potentially risky helpers (such as any that interact with the file system)
- To increase the number of test cases and resulting test coverage
```

> > More than 130 Handlebars helpers in ~20 categories. Helpers can be used with Assemble, Generate, Verb, Ghost, gulp-handlebars, grunt-handlebars, consolidate, or any node.js/Handlebars project.

Reviewer: ```suggestion
```

Install with [npm](https://www.npmjs.com/):

```sh
$ npm i handlebars-kit
```

Install with [yarn](https://yarnpkg.com):

```sh
$ yarn add handlebars-kit
```

## Usage

The main export returns a function that needs to be called to expose the object of helpers.

**Get all helpers**

```js

> var helpers = require('handlebars-kit')();

Reviewer: ```suggestion
const helpers = require('handlebars-kit')();
```

Helper collections are exposed as getters, so only the helpers you want will be required and loaded.

```js

> var helpers = require('handlebars-kit');
> var math = helpers.math();

Reviewer: ```suggestion
const helpers = require('handlebars-kit');
const math = helpers.math();
```

//=> only the `math` helpers

> var helpers = require('handlebars-kit');
> var array = helpers.array();

Reviewer: ```suggestion
const helpers = require('handlebars-kit');
const array = helpers.array();
```

Helper collections are exposed as getters, so only the helpers you want will be required and loaded.

```js

> var helpers = require('handlebars-kit')(['math', 'string']);

Reviewer: ```suggestion
const helpers = require('handlebars-kit')(['math', 'string']);
```

//=> only the `math` and `string` helpers
```

**Optionally pass your own handlebars**

```js
var handlebars = require('handlebars');

> var helpers = require('handlebars-kit')({

Reviewer: ```suggestion
const helpers = require('handlebars-kit')({
```

### [array helpers](#array)

Visit the: [code](lib/array.js) | [unit tests](test/array.js)

> * **[after](#after)** ([code](lib/array.js#L22) | [tests](test/array.js#L13))

Reviewer: Are these links to the code line numbers (ie, `lib/array.js#13`) still correct, or have they changed since the fork/refactoring?
